packages/transcript-view/test/transcript-view.test.js

```diff
@@ -16,6 +16,41 @@ describe('TranscriptView', () => {
     expect(el.config).to.equal(undefined);
   });
 
+  it('displays context zones if enabled', async () => {
+    const el = await fixture(html`
+      <transcript-view
+        showContextZones='true'>
+      </transcript-view>
```

Comment:
QUESTION: You okay with the `<transcript-view>` start and end tags containing a newline? (Same question applies below.)

Reply:
@bfalling Are you talking about the parameters being on separate lines or the opening and closing tags being on separate lines or something else?

I mostly did it for readability.. once you start passing a few parameters into the component, it starts to get pretty dense so the line breaks just helps visually distinguish everything.